charts/opentelemetry-demo/values.yaml

```diff
@@ -620,7 +620,7 @@ opentelemetry-collector:
           insecure: true
       # Create an exporter to Prometheus (metrics)
       prometheus:
-        endpoint: "${env.MY_POD_IP}:9464"
+        endpoint: ":9464"
```

Comment:
How does this endpoint work? Is the prom chart doing some magic behind the scenes?